packages/editor/src/pages/Editor/store/misc/selectors.ts

```diff
@@ -0,0 +1,10 @@
+import { createSelector } from 'reselect';
+import { Utilities, HostType, PlatformType } from '@microsoft/office-js-helpers';
+import { isPoppedOut } from 'common/lib/utilities/popout.control';
+
+import { selectors as host } from 'script-lab-core/lib/modules/host';
+
+export const getIsRunnableOnThisHost = createSelector(
```

Comment:
This needed to be added due to the fact that the functionality used to exist in the local host selectors, but doesn't in script-lab-core's host selectors.